.goreleaser.yml

```diff
@@ -63,6 +63,7 @@ release:
     ## Installation
 
     ```bash
+    curl -L -s https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.24.0/install.sh | bash -s v0.24.0
```

Comment:
Should we make the OLM version an environment variable that comes in from the Makefile?

Reply:
> Should we make the OLM version an environment variable that comes in from the Makefile?

I think that's a good idea if we plan to depend on OLMv0 long term